unlocks.csv

```diff
@@ -403,34 +386,42 @@ Analysis|PlasteelIngot|0,0,PlasteelIngot
 Analysis|Beacon|0,0,Beacon
 Analysis|Gravsphere|0,0,Gravsphere
 Analysis|LaserCutter|0,0,LaserCutter
+Goal|OnCraftBattery|0,1,Battery
+Goal|OnCraftComputerChip|0,1,ComputerChip|CopperWire
+Goal|OnCraftTank|0,1,Tank
+Goal|OnCraftDoubleTank|0,1,DoubleTank|Glass|Tank
+Goal|UnlockHatchingEnzymes|0,1,HatchingEnzymes
+Goal|UnlockRadiationSuit|0,1,
+Goal|UnlockRadiationSuit|1,1,
+Goal|UnlockRadiationSuit|2,1,
+Goal|Aurora_RingRoom_Terminal2|0,1,Welder|LaserCutter|Battery
```

Comment:
You don't need the propulsion cannon for the Aurora access, you can just take the top entrance instead. And even without the radiation suit you can get all the way through to the captains quarters with just a few medkits.